src/connection/Connection.ts

```diff
@@ -305,6 +305,18 @@ export class Connection {
         await migrationExecutor.undoLastMigration();
     }
 
+    /**
+     * Runs all pending migrations.
```

Comment:
Could you please update or remove this comment ? :)

Reply:
Done, good spot